src/studio/nuxeo-studio/nuxeo-studio-release-notes/2018-06-11-studio-73-14.md

```diff
@@ -0,0 +1,88 @@
+Your private packages can now be added as dependencies in your application definition. This is particularly useful when using [custom packages delivered through the Nuxeo Markerplace]({{page page='delivering-a-customization-package-through-the-nuxeo-marketplace'}}#example-use-cases).
+
+Note that it is not yet possible to choose which version of a private package you want to depend on. Application definition will always add a dependency to the latest available version on the marketplace every time you save your changes.
+
+As with public packages, Nuxeo Studio will automatically [load their content]({{page page='2018-05-28-studio-73-13'}}#automatic-dependencies-registries-loading) when possible. For that to happen:
+- Make sure your project's `pom.xml` depends on a Nuxeo distribution version of 9.10 or newer
+- In your marketplace package's `assembly.xml` file, add the following instruction before generating the final zip:
+
+```
+<nx:studioExtraction todir="${outdir}/marketplace">
+  <fileset dir="${outdir}/marketplace/install/bundles"/>
+</nx:studioExtraction>
+```
+
+Here is an extract of the final result:
+
+```
+[...]
+    <nx:studioExtraction todir="${outdir}/marketplace">
+      <fileset dir="${outdir}/marketplace/install/bundles"/>
+    </nx:studioExtraction>
+
+    <zip destfile="${outdir}/${maven.project.artifactId}-${maven.project.version}.zip" basedir="${outdir}/marketplace"/>
+    <artifact:attach file="${outdir}/${maven.project.artifactId}-${maven.project.version}.zip" type="zip"/>
+  </target>
+</project>
+```
+
+Note that the above example assumes you have been generating your package using our recommended tool: [nuxeo-cli]({{page space='nxdoc' page='nuxeo-cli'}}). Otherwise you will need to adapt these instructions to your own use case.
+
+## Token Generation
+This release introduces the ability to generate personal authentication tokens from the Nuxeo Online Services portal.
+
+Tokens can be used as a replacement for a password, and can be revoked anytime. A few situations where they can be leveraged include:
+- In your continuous integration / continuous delivery chain, to have your software authenticate against our Maven private repository,
+- When you authenticate using a third party provider (Google or Okta) and need to use our command line tools with options that require a password to be provided, for instance to register an instance using nuxeoctl, link your Studio project to your Java development one in nuxeo-cli.
+
+To generate a token,
+- Login to <a target="_blank" href="https://connect.nuxeo.com">Nuxeo Online Services</a>
+- Visit the `My Tokens` tab
+- Create your token using the corresponding button and provide a clear name for it (ex: "CI Chain")
+![]({{file name='token-management.png'}} ?border=true,w=650)
+
+Your token will only be shown once, so make sure to save it. If you ever forget it or think it may have been compromised, you can revoke it anytime and generate a new one.
```

Comment:
Your token will only be shown once, so be sure to save it in a secure place. If you ever forget it or think it may have been compromised, you can revoke it anytime and generate a new one.